src/components/DataTable/_Header.js

```diff
@@ -30,13 +29,7 @@ class Header extends React.Component {
       <td onClick={this.handleClick} className={className} title={title}>
         {children}{' '}
         {order &&
-          sortable && (
-            <i
-              className={
-                order === 'asc' ? 'fa fa-caret-up' : 'fa fa-caret-down'
-              }
-            />
-          )}
+          sortable && <i className={order === 'desc' ? 'fa fa-caret-up' : 'fa fa-caret-down'} />}
```

Comment:
There doesn't seem a common consensus online which way is better. What's _your_ reasoning for seeing the previous version as incorrect?

Both ways have their merits I guess

- `asc ▲` small end vs big end
- `asc ▼` is pointing in the direction to read the list

since `asc` is usually the preferred way to read a list, having a triangle pointing "backwards" for ascending conveys to me that there might be something off while it isn't.